### PR TITLE
[DOC] Update textCNN results 

### PR DESCRIPTION
**Pull request by xiaotinghe**
## Description ##
Update textCNN results 
### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Update textCNN results

### CODE REVIEW

**Comment by xiaotinghe** [comment]
done

**Comment by mli** [comment]
Job PR-737/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-737/2/index.html

File: scripts/sentiment_analysis/index.rst [hunk 1]
```diff
@@ -48,31 +48,31 @@ epoch:
...
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
-| static         |   79.9   |   46.6    |   86.9    |   92.0   |   91.4   |   82.9   |   89.4   |
+| static         |   79.4   |   48.1    |   87.1    |   91.8   |   91.4   |   83.0   |   89.6   |
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
-| non-static     |   80.2   |   46.1    |   86.5    |   92.4   |   92.4   |   83.4   |   89.4   |
+| non-static     |   80.0   |   47.0    |   85.6    |   91.9   |   92.4   |   82.9   |   89.2   |
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
-| multichannel   |   80.2   |   47.9    |   86.0    |   92.9   |   92.8   |   83.6   |   89.8   |
+| multichannel   |   80.0   |   48.1    |   85.8    |   92.1   |   93.8   |   83.3   |   89.6   |
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
```
Comment (astonzhang) [inline scripts/sentiment_analysis/index.rst]: according to https://github.com/dmlc/web-data/blob/master/gluonnlp/logs/sentiment/TREC_multichannel.log
test acc for TREC-multichannel is 93.2

according to https://github.com/dmlc/web-data/blob/master/gluonnlp/logs/sentiment/TREC_non-static.log
test acc for TREC-non-static is 93.2

Can you double-check all the 28 numbers to make sure they are consistent with the uploaded logs?

**Comment by mli** [comment]
Job PR-737/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-737/3/index.html

**Comment by mli** [comment]
Job PR-737/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-737/4/index.html

File: scripts/sentiment_analysis/index.rst (hunk 1 above)
Comment (xiaotinghe) [reply scripts/sentiment_analysis/index.rst]: has been corrected.

**Comment by yzh119** [comment]
Hi, I noticed a performance gap between previous results: https://github.com/dmlc/gluon-nlp/blob/5e1e7155517d0bed020b76357ff1bd77cedaf45d/scripts/sentiment_analysis/index.rst and results in this pr.
Could I know what causes the difference?

**Comment by astonzhang** [comment]
> Hi, I noticed a performance gap between previous results: https://github.com/dmlc/gluon-nlp/blob/5e1e7155517d0bed020b76357ff1bd77cedaf45d/scripts/sentiment_analysis/index.rst and results in this pr.
> Could I know what causes the difference?

We updated the data sets.

**Pull request merged**